src/Python/Meshes/ColoredElevationMap.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+import vtk
+
+def main():
+
+    # Create a grid points
+    points = vtk.vtkPoints()
+    GridSize = 20;
+    xx=0.0 
+    yy=0.0
+    zz=0.0
+    for x in range(0, GridSize):
```

Comment:
Please use `vtkMinimalStandardRandomSequence` here in place of `vtkMath.Random`. See: [OrientedCylinder](https://lorensen.github.io/VTKExamples/site/Python/GeometricObjects/OrientedCylinder/) for an example using `vtkMinimalStandardRandomSequence`.

Reply:
Thank you. I added this request.